mllib/src/test/scala/org/apache/spark/ml/feature/StringIndexerSuite.scala

```diff
@@ -331,4 +357,51 @@ class StringIndexerSuite
     val dfWithIndex = model.transform(dfNoBristol)
     assert(dfWithIndex.filter($"CITYIndexed" === 1.0).count == 1)
   }
+
+  test("StringIndexer multiple input columns") {
```

Comment:
Is there a test case, for testing `frequencyAsc/Desc`, when frequency equal, it will secondary sort by alphabets ?

Reply:
Good point. I added one test for this.